src/main/java/net/fabricmc/loom/LoomGradlePlugin.java

```diff
@@ -40,37 +41,43 @@
 import net.fabricmc.loom.decompilers.DecompilerConfiguration;
 import net.fabricmc.loom.task.LoomTasks;
 
-public class LoomGradlePlugin implements Plugin<Project> {
+public class LoomGradlePlugin implements Plugin<PluginAware> {
 	public static boolean refreshDeps;
 	public static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
 	public static final ObjectMapper OBJECT_MAPPER = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
 
 	@Override
-	public void apply(Project project) {
-		project.getLogger().lifecycle("Fabric Loom: " + LoomGradlePlugin.class.getPackage().getImplementationVersion());
+	public void apply(PluginAware target) {
```

Comment:
The diff can be simplified by retaining `apply(Project)` and adding a `apply(PluginAware)` at the same time. The override will move to the `apply(PluginAware)`, which will be:
```java
	@Override
	public void apply(PluginAware target) {
		target.getPlugins().apply(LoomRepositoryPlugin.class);
		if (target instanceof Project) {
			apply((Project) target);
		}
	}
```